tests/data.json

```diff
@@ -231,7 +255,8 @@
         "analyte_type": "Virtual",
         "method_of_sample_procurement": "Autopsy",
         "preservation_method": "Frozen",
-        "specimen_status": "Not Available"
+        "specimen_status": "Not Available",
+        "cell_entity": "whole cell"
       },
       "invalid": [
```

Comment:
I think the reason why the tests didn't catch the typos were bc that code wasn't being tested. For every field that has a validator, can you add an entry to the `invalid` list so that it gets tested?